client/src/pages/OnBoarding/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useRef } from "react";
+import { Paper, Grid, Button, makeStyles } from "@material-ui/core";
+
+import SetTimezoneUrl from "./SetTimezoneUrl";
+import ConnectGoogleCalendar from "./ConnectGoogleCalendar";
+import SetAvailability from "./SetAvailability";
+
+const OnBoarding = (props) => {
+  //styles
+  const classes = useStyles();
+
+  //data related variables
+  const [pageNum, setPageNum] = useState(0);
+  const url = useRef();
```

Comment:
It would be easier to keep these as state objects and make handler functions that get passed down to the children, rather than worrying about keeping high order refs consistent with lower level state